tidb-cloud/serverless-driver.md

```diff
@@ -281,27 +287,37 @@ The type mapping between TiDB Serverless and Javascript is as follows:
 | DECIMAL              | string          |
 | CHAR                 | string          |
 | VARCHAR              | string          |
-| BINARY               | string          |
-| VARBINARY            | string          |
+| BINARY               | Uint8Array      |
+| VARBINARY            | Uint8Array      |
 | TINYTEXT             | string          |
 | TEXT                 | string          |
 | MEDIUMTEXT           | string          |
 | LONGTEXT             | string          |
-| TINYBLOB             | string          |
-| BLOB                 | string          |
-| MEDIUMBLOB           | string          |
-| LONGBLOB             | string          |
+| TINYBLOB             | Uint8Array      |
+| BLOB                 | Uint8Array      |
+| MEDIUMBLOB           | Uint8Array      |
+| LONGBLOB             | Uint8Array      |
 | DATE                 | string          |
 | TIME                 | string          |
 | DATETIME             | string          |
 | TIMESTAMP            | string          |
 | ENUM                 | string          |
 | SET                  | string          |
-| BIT                  | string          |
+| BIT                  | Uint8Array      |
 | JSON                 | object          |
 | NULL                 | null            |
 | Others               | string          |
 
+> **Note:**
+>
+> Make sure to use the default `utf8mb4` charset in TiDB Serverless for the type conversion to JavaScript strings, because TiDB Cloud serverless driver uses the UTF-8 encoding to decode them to strings. 
+
+> **Note:**
+>
+> TiDB Cloud serverless driver data type mapping changes:
+> 
+> - v0.1.0: the `BINARY`、`VARBINARY`、`TINYBLOB`、`BLOB`、`MEDIUMBLOB`、`LONGBLOB`, and `BIT` types are now returned as a `Uint8Array` instead of a `string`.
```

Comment:
```suggestion
> - v0.1.0: the `BINARY`, `VARBINARY`, `TINYBLOB`, `BLOB`, `MEDIUMBLOB`, `LONGBLOB`, and `BIT` types are now returned as a `Uint8Array` instead of a `string`.
```